### DISCUSSION TITLE
Bticino, some issues.

### DISCUSSION THREAD

**Discussion by drubens** [Koenkk/zigbee2mqtt#10084] [Devices]
Hallo Dears
I've playing with bticino tools and everything is working perfectly on baseline functions.

I've just some issues.
- The dimmer K4411C seems not working on brightness functions. I can switch on and off but dimmer doesn't work.
- all tools as K4411C and L4531C could show a blue led once they are active but I'm not sure how and if it could be configured with zigbee2mqtt.

No issues on energy meter. Works like a sword.

I'm using HA with a slae.sh CC2652RB

I'm not an expert, be patience with me and thanks in advance for any feedback.

Ciao
R

**Comment by AndreaNizzola** [comment]
Hi Drubens, I'm having the same issue with the K4411C, have you been able to solve the problem? I can't find a solution online.
Thankyou,
Andrea

**Reply by drubens** [reply]
Ciao Andrea. I've opened an issue about that.
https://github.com/Koenkk/zigbee2mqtt/issues/10580
I did something suggested but nothing happening. Please have a look and let me know your experience. 
Grazie

**Reply by AndreaNizzola** [reply]
Hey mate, I got it working, With MQTT you need to send the command:
`{
    "dimmer_enabled": "ON"
}`

This is the page where I found the information: https://www.zigbee2mqtt.io/devices/067771.html
Also thankyou as I found this page from the information I found on your support ticket.

**Reply by drubens** [reply]
Great. Could you give me a bit of instruction to setup it on HA? I'm not expert but I learn fast if I see :)
It would be great.

**Reply by drubens** [reply]
Found that. I was not realised that on mqtt now I see new features for those lights. 
Thanks ciao

**Reply by AndreaNizzola** [reply]
Cool, did you get it working or do you still need a hand?
BTW are you italian?

**Reply by drubens** [reply]
Yes on both questions :)
Ho chiuso il ticket con uno screenshot. Ora si vedono le ozioni necessarie nella UI di Zigbee2mqtt. 
Top!

**Reply by AndreaNizzola** [reply]
Perfetto, grazie di nuovo, senza il ticket non avrei avuto nessuna idea di cosa cercare

**Comment by AndreaNizzola** [comment]
Ciao, ho un problema con gli interruttori, si connettono senza problemi e funzionano come dovrebbero pero' dopo un po' di tempo che sono dimmati iniziano a lampeggiare, come se la luminosita' incrementa al 100% per una frazione di secondo. A volte lo fa per pochi secondi ma altre continua per interi minuti. Tutti gli interruttori lo fanno, tu hai avuto un problema simile?Grazie,Andrea
    Il mercoledì 2 febbraio 2022, 22:39:47 GMT+11, drubens ***@***.***> ha scritto:  
 
 


Yes on both questions :)
Ho chiuso il ticket con uno screenshot. Ora si vedono le ozioni necessarie nella UI di Zigbee2mqtt.
Top!

—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Reply by drubens** [reply]
ciao, hai le stesse lampadine su tutti? lo fanno tutte le lampadine o solo alcune? proverò a lasciare qualche luce soffusa per tanto tempo e vedo se replico il problema. Avevo avuto in passato un problema, attenuando la luminosità arrivavo quasi alla percentuale minima e la luce riprendeva luminosità per una frazione di secondo. Anche in fase di spegnimento ed accensione faceva questa cosa. Si è sistemato con una versione firmware nuova di bticino ma erano ancora collegate al gateway Bticino. Per ora con HA e zigbee2mqtt con chiavetta usb, tutto sembra stabile.

**Comment by AndreaNizzola** [comment]
Si, ho gli stassi faretti in tutte le stanze, però ho preso dei faretti apposta per essere soffusi. Funzionavano senza problemi con il varialuce montato prima. Oggi provo a connetere un interruttore al gateway originale e vedere se funziona.Message ID: ***@***.***>
